HintList.py

```diff
@@ -998,10 +998,10 @@ def tokens_required_by_settings(world):
     'Fire Temple GS Scarecrow Top':                                ("a #spider-friendly scarecrow# atop a volcano hides", "a #spider-friendly scarecrow# atop the Fire Temple hides", 'exclude'),
     'Fire Temple GS Scarecrow Climb':                              ("a #spider-friendly scarecrow# atop a volcano hides", "a #spider-friendly scarecrow# atop the Fire Temple hides", 'exclude'),
 
-    'Fire Temple MQ GS Above Fire Wall Maze':                      ("a #spider above a fiery maze# holds", None, 'exclude'),
-    'Fire Temple MQ GS Fire Wall Maze Center':                     ("a #spider within a fiery maze# holds", None, 'exclude'),
+    'Fire Temple MQ GS Above Flame Maze':                          ("a #spider above a fiery maze# holds", None, 'exclude'),
+    'Fire Temple MQ GS Flame Maze Center':                         ("a #spider within a fiery maze# holds", None, 'exclude'),
     'Fire Temple MQ GS Big Lava Room Open Door':                   ("a #Goron trapped near lava# befriended a spider with", None, 'exclude'),
-    'Fire Temple MQ GS Fire Wall Maze Side Room':                  ("a #spider beside a fiery maze# holds", None, 'exclude'),
+    'Fire Temple MQ GS Flame Maze Side Room':                      ("a #spider beside a fiery maze# holds", None, 'exclude'),
```

Comment:
Not a huge deal either way, but what's exactly the reasoning for some of these name changes? Would rather avoid them if possible to not unnecessarily break existing plandos

Reply:
> 

Ok I'll try and get the rebase done tonight.

Reply:
I renamed these for consistency with other checks in the area that were all called 'flame maze'. I think all of these checks with that name are new checks however, so you could opt to rename all of those instead. There was a preexisting logic trick also with the internal name "flame_maze". I added some regions that used the 'flame maze' name as well that might as well be updated if you do go for that name change.

Below, I also renamed the shadow invisible blades GS to match with other checks in the room. I think the added naming consistency there probably is worth it.

Reply:
Fair enough, I'll take consistency as a good reason for renaming